spec/requests/items_requests_spec.rb

```diff
@@ -190,5 +190,36 @@
         end
       end
     end
+
+    describe 'GET #show' do
+      let!(:item_category) { create(:item_category, name: 'CURRENTCATEGORY') }
+      let!(:item) { create(:item, organization: organization, name: "ACTIVEITEM", item_category_id: item_category.id, distribution_quantity: 1, on_hand_recommended_quantity: 1, package_size: 3) }
+      it 'shows complete item details except custom request' do
+        get item_path(id: item.id)
+        expect(response.body).to include('Name')
+        expect(response.body).to include(item.name)
+        expect(response.body).to include('Base Item')
+        expect(response.body).to include(item.base_item.name)
+        expect(response.body).to include('Category')
+        expect(response.body).to include(item.item_category.name)
+        expect(response.body).to include('Value Per Item')
+        expect(response.body).to include(item.value_in_cents&.to_s)
+        expect(response.body).to include('Quantity Per Indivudual')
+        expect(response.body).to include(item.distribution_quantity&.to_s)
+        expect(response.body).to include('On hand minimum quantity')
+        expect(response.body).to include(item.on_hand_recommended_quantity&.to_s)
+        expect(response.body).to include('On hand recommended quantity')
+        expect(response.body).to include(item.on_hand_minimum_quantity&.to_s)
```

Comment:
To avoid later developer confusion,  I would reorder these lines so that the variable is being checked adjacent to the label.